Queueomatic/Server/Endpoints/ForgetPassword/ForgetPasswordRequest.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace Queueomatic.Server.Endpoints.ForgetPassword;
+
+public record ForgetPasswordRequest(string Email);
```

Comment:
This request needs to be validated for preventing any waste of sources in case of the wrong email type.